contrib/qutebrowser/template.py

```diff
@@ -32,13 +26,28 @@
     "qt6-qtbase-sql",
     "qt6-qtwebengine",
 ]
+checkdepends = depends + [
+    "python-beautifulsoup4",
+    "python-hypothesis",
+    "python-pytest",
+    "python-pytest-bdd",
+    "python-pytest-benchmark",
+    "python-pytest-instafail",
+    "python-pytest-mock",
+    "python-pytest-qt",
+    "python-pytest-rerunfailures",
+    "python-soupsieve",
+    "python-vulture",
+    "xwayland-run",
+    "xserver-xorg-xvfb",
+]
 pkgdesc = "Keyboard driven web browser with a minimalist gui"
 maintainer = "ttyyls <contact@behri.org>"
 license = "GPL-3.0-only"
 url = "https://qutebrowser.org"
 source = f"https://github.com/qutebrowser/qutebrowser/archive/refs/tags/v{pkgver}.tar.gz"
-sha256 = "9307d55f1d9157b7906ac08566e3951bffd6f5b753432deaa9a13681995ba3ca"
-# unpackaged dependencies
+sha256 = "6558bc55bdd7d7a5cefbb9166eea14cd7eaa53ba3e97f6814eb3ebaa548f68e2"
+# XXX: $DISPLAY?
```

Comment:
this is because make_check_wrapper does not work for pep517 but otherwise it would